scalding-core/src/main/scala/com/twitter/scalding/typed/Grouped.scala

```diff
@@ -96,16 +87,9 @@ object Grouped {
   def apply[K, V](pipe: TypedPipe[(K, V)])(implicit ordering: Ordering[K]): Grouped[K, V] =
     IdentityReduce(ordering, pipe, None)
 
-  def keySorting[T](ord: Ordering[T]): Fields = sorting("key", ord)
-  def valueSorting[T](implicit ord: Ordering[T]): Fields = sorting("value", ord)
-
-  def sorting[T](key: String, ord: Ordering[T]): Fields = {
-    val f = new Fields(key)
-    val comparator: Comparator[_] = ord match {
-      case bufOrd: OrderedSerialization[_] => new CascadingBinaryComparator(BoxedKeyBinary(bufOrd))
-      case nonBinary => nonBinary
-    }
-    f.setComparator(key, comparator)
+  def valueSorting[V](ord: Ordering[V]): Fields = {
```

Comment:
we should merge this with `singleOrdered` to reduce code duplication `valueSorting == singleOrdered("value")`